cpp/src/io/comp/nvcomp_adapter.cpp

```diff
@@ -152,6 +160,25 @@ size_t batched_decompress_temp_size(compression_type compression,
   return temp_size;
 }
 
+void check_is_zstd_enabled()
```

Comment:
Probably this should be put into an anonymous namespace.

Reply:
hm, there's many more functions here that could be in an anonymous namespace. If you don't mind, I would prefer to make this change in a future PR.

Reply:
Sure.